### DISCUSSION TITLE
Option to assign Fan + Power mode to switch between multiple options if plugged into power or not.

### DISCUSSION THREAD

**Discussion by just1fye** [seerge/g-helper#2290] [Ideas]
Basically my idea is, similar to optimized mode that swaps between dGPU and iGPU you can have a option to assign a regular fan + power config or a custom one to be used when either plugged in or on battery automatically. For example when on pd it automatically goes to a custom config assigned but when on battery it automatically swaps to silent or any other option selected by the user.

**Comment by seerge** [comment]
Application is already remembers last used performance mode on battery / plugged in and switches automatically. Please check readme https://github.com/seerge/g-helper?tab=readme-ov-file#gear-automation
